src/messages.h

```diff
@@ -0,0 +1,14 @@
+#ifndef MESSAGES_H
```

Comment:
Please do if guards in a consistent manner, like the other headers. For example, `SOURCEMETA_JSONSCHEMA_CLI_MESSAGES_H_`